ci/docs/build.sh

```diff
@@ -25,10 +25,43 @@ nvidia-smi
 
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
-conda activate rapids
+
+conda install -c conda-forge conda-build -y
+
+# We don't use 'rapids' conda environment here.
+# cuCIM depends on CuPy 9 which should be adopted in 0.20
+LIBCUCIM_BLD_PATH=$WORKSPACE/ci/artifacts/cucim/cpu/.conda-bld
+CUCIM_BLD_PATH=/opt/conda/envs/rapids/conda-bld
+mkdir -p ${CUCIM_BLD_PATH}
+
+gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge/label/cupy_rc -c conda-forge -c rapidsai-nightly \
+    --python=${PYTHON_VER} \
+    --dirty \
+    --no-remove-work-dir \
+    --croot ${CUCIM_BLD_PATH} \
+    conda/recipes/cucim
+
```

Comment:
is this build relatively quick? I'm assuming you're choosing to build here vs. install from conda so that the docs are using the latest source code, is that correct? just trying to make sure we have the minimum required environment necessary for doc builds.

Reply:
that's a good point @ajschmidt8 i think we can be slightly out of sync and we can pull from anaconda.org